daemon/errors.go

```diff
@@ -133,7 +133,8 @@ func translateContainerdStartErr(cmd string, setExitCode func(int), err error) e
 	if contains(errDesc, cmd) &&
 		(contains(errDesc, "executable file not found") ||
 			contains(errDesc, "no such file or directory") ||
-			contains(errDesc, "system cannot find the file specified")) {
+			contains(errDesc, "system cannot find the file specified") ||
+			contains(errDesc, "failed to run runc create/exec call")) {
```

Comment:
Is this error only created because of an invalid _container_ process? (Couldn't find where this error is generated after a quick search; just want to be sure we're not masking possible issues on the host)

Reply:
Ah, it's returned in the LCOW case from opengcs: https://github.com/Microsoft/opengcs/blob/master/service/gcs/runtime/runc/runc.go#L580